mollie-payments-for-woocommerce/includes/mollie/wc/plugin.php

```diff
@@ -337,7 +364,15 @@ public static function addGateways( array $gateways ) {
 
 		$gateways = array_merge( $gateways, self::$GATEWAYS );
 
-		// Return if function get_current_screen() is not defined
+        isset($_POST['post_data']) and parse_str($_POST['post_data'], $postData);
```

Comment:
Superglobal access. This is not great. If we cannot use a standard request, I suggest [`filter_input()`][].

[`filter_input()`]: https://www.php.net/manual/en/function.filter-input.php

Reply:
@XedinUnknown That's actually not a problem because there's no used values, just a check for an isset. see code below.